dtfs-central-api/src/v1/controllers/utilisation-report-service/get-selected-fee-records-details.controller/index.ts

```diff
@@ -58,11 +59,14 @@ export const getSelectedFeeRecordDetails = async (req: GetSelectedFeeRecordDetai
 
     const canAddToExistingPayment = await canFeeRecordsBeAddedToExistingPayment(reportId, selectedFeeRecords);
 
+    const gbpTolerance = await PaymentMatchingToleranceService.getGbpPaymentMatchingTolerance();
```

Comment:
Should we/do we need to throw an error here if no `gbpTolerance` is found? Other checks above throw an error

Reply:
`PaymentMatchingToleranceService.getGbpPaymentMatchingTolerance` will either return a value, or throw an error. It is not possible for it to just not return a value. If it throws because there is no tolerance, then that gets caught by the try catch block surrounding this block of code.

Reply:
Makes sense, thanks @RLavender98